src/Simulator/IncrementalRedstoneSimulator/RedstoneHandlerFactory.h

```diff
@@ -0,0 +1,19 @@
+
+#pragma once
+
+#include "World.h"
+#include "Chunk.h"
+#include "IncrementalRedstoneSimulator.h"
+
+
+
+
+
+class cRedstoneHandlerFactory
+{
+public:
+	static ComponentPtr GetComponent(cWorld & a_World, BLOCKTYPE a_BlockType, cIncrementalRedstoneSimulatorChunkData * a_Data);
```

Comment:
No, I don't want large classes. Free functions are not in classes, so do not result in large classes.